docs/HowToGuides.md

```diff
@@ -1,5 +1,63 @@
 # How To ...
 
+### Create a multi-step PR
+
+Big PRs (i.e. PRs touching more than 100 lines) are hard to review. 
+Using the _multi-step PR_ format explained below allows the review to be done incrementally.
+
+1. Dev creates the PR branch consisting of a series of commits corresponding to a sequence of logical steps, 
+   each step advancing the code base a step towards the end goal of the PR. The build should pass at each step.
+   <details>
+   <summary>Instructions</summary>
+   
+   * There Ideal approach for creating a multi-step commit sequence: 
+     * Plan the PR as a sequence of logical steps.
+     * Implement each step, committing each step as a separate commit. 
+       * Write detailed commit messages as if you were explaining your commit to a future developer.
```

Comment:
"detailed commit messages" is a different matter to "proper commit organization", I think. While it is good to have both, it is possible to have proper commit organization without detailed commit messages and vice versa.

To keep this section as succinct as possible, I think we could have a passing mention on writing good commit messages as well, but I don't think we need to dedicate a whole point and a blockquote towards it. (e.g. `Implement each step, committing each step as a separate commit. ( [Remember to write detailed commit messages!](writing-proper-commit-messages.md))`

To elaborate: "proper commit organization" mainly focuses on the *diff* of each commit. Separating logical changes into different commits allows reviewers to view the diffs of different logical changes in isolation. Detailed commit messages certainly help to *communicate* a change to the reviewer, but imo it is not strictly required for a PR to be considered a proper "multi-step PR".